pkg/config/configmanager.go

```diff
@@ -15,27 +13,17 @@ import (
 // ConfigFileManagerOpts describes how to configure the ConfigFileManager for
 // serving configuration files
 type ConfigFileManagerOpts struct {
-	// BucketStoreConfig is the local file location for the configuration used to
-	// write and read configuration data to/from the bucket. The format of this
-	// configuration file should be compatible with storage.NewBucketStorage
-	BucketStoreConfig string
 
 	// LocalConfigPath provides a backup location for storing the configuration
 	// files
 	LocalConfigPath string
 }
 
-// IsBucketStorageEnabled returns true if bucket storage is enabled.
-func (cfmo *ConfigFileManagerOpts) IsBucketStorageEnabled() bool {
-	return cfmo.BucketStoreConfig != ""
-}
-
 // DefaultConfigFileManagerOpts returns the default configuration options for the
 // config file manager
 func DefaultConfigFileManagerOpts() *ConfigFileManagerOpts {
 	return &ConfigFileManagerOpts{
-		BucketStoreConfig: "",
-		LocalConfigPath:   "/",
+		LocalConfigPath: "/",
 	}
 }
```

Comment:
I think we can remove all of this now that ConfigFileManager accepts the `storage.Storage`